chapter01/sxg/HelloWorld/src/main/java/com/github/sebin/HelloWorldController.java

```diff
@@ -0,0 +1,23 @@
+package com.github.sebin;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.servlet.ModelAndView;
+ 
+@Controller
+public class HelloWorldController {
+	String message = "Welcome to Spring MVC!";
+ 
+	@RequestMapping("/hello")
+	public ModelAndView showMessage(
+			@RequestParam(value = "name", required = false, defaultValue = "World") String name) {
+		System.out.println("in controller");
```

Comment:
It looks fine but the brief was to load the name through spring rather than pass it in a parameter.